index.html

```diff
@@ -2779,42 +2838,103 @@ <h3>Specifying sort and search functionality</h3>
 <aside class="links" id="links_char_sort">
 <p class="links_title">Useful background and overviews for this section</p>
 <ul>
-<li class="w3"><p class="link"><a href="https://www.w3.org/TR/charmod/#sec-CollationUnits">Units of collation</a>, in <cite>Character Model for the World Wide Web: Fundamentals</cite>.</p></li>
+	<li><p class="link">Unicode's <a href="https://github.com/unicode-org/icu/blob/main/docs/userguide/collation/index.md">ICU Collation User Guide</a></p></li>
+	<li><p class="link"><a href="https://unicode.org/reports/tr10">Unicode Collation Algorithm</a> [[UTS10]]</p></li>
+    <li class="w3"><p class="link"><a href="https://www.w3.org/TR/charmod/#sec-CollationUnits">Units of collation</a>, in <cite>Character Model for the World Wide Web: Fundamentals</cite>.</p></li>
 </ul>
 </aside>
 
-
-    <p>Applications often need to organize sets of information or content. Frequently this involves sorting the content so that users can find what they are looking for. Many data types, such as numbers or dates, are easily sorted by comparing the values. When it comes to textual information, however, the nature of character encodings brings some additional complexity.</p>
+    <p>Applications often need to organize sets of information or content. Frequently this involves sorting the content. Many data types, such as numbers or dates, are easily sorted by comparing the values. When it comes to textual information, however, the nature of character encodings and user expectations regarding "alphabetical" order brings some additional complexity.</p>
```

Comment:
Numbers and dates are only easy to sort if you compare internal representations of the data.

Reply:
Done with:

```suggestion
Many non-textual data types, such as numbers or dates, can be easily sorted using the internal data representation. 
```